capacity4more/modules/c4m/features_og/c4m_features_og_events/c4m_features_og_events.module

```diff
@@ -190,3 +199,56 @@ function c4m_features_og_events_facet_items_alter(&$build, &$settings) {
     }
   }
 }
+
+/**
+ * Implements hook_entity_info_alter().
+ *
+ * Add "Upcoming event" view mode.
+ */
+function c4m_features_og_events_entity_info_alter(&$entity_info) {
+
+  $entity_info['node']['view modes']['upcoming_event'] = array(
+    'label' => t('Upcoming event'),
+    'custom settings' => TRUE,
+  );
+}
+
+/**
+ * Fetches upcoming events (Events in the future that haven't started yet),
+ * Depending on the user's role (guest, logged-in),
+ * This function fetches the events from user's groups or public groups.
+ * 1 Event for anonymous user, 3 for logged in.
+ *
+ * @return string
+ *  HTML of rendered upcoming events.
+ */
+function c4m_features_og_events_upcoming_events() {
+  global $user;
+  $date = date('Y-m-d H:i:s', time());
```

Comment:
I think it's the upcoming event, no matter when it is except for ones that
already started, we need to review the wiki together tomorrow.

On Monday, February 9, 2015, HelenaEksler <notifications@github.com> wrote:

> In
> capacity4more/modules/c4m/features_og/c4m_features_og_events/c4m_features_og_events.module
> <https://github.com/capacity4dev/capacity4more/pull/463#discussion_r24338007>
> :
>
> > +    'custom settings' => TRUE,
> > +  );
> > +}
> > +
> > +/**
> > + * Fetches upcoming events (Events in the future that haven't started yet),
> > + * Depending on the user's role (guest, logged-in),
> > + * This function fetches the events from user's groups or public groups.
> > + * 1 Event for anonymous user, 3 for logged in.
> > + *
> > + * @return string
> > + *  HTML of rendered upcoming events.
> > + */
> > +function c4m_features_og_events_upcoming_events() {
> > +  global $user;
> > +  $date = date('Y-m-d H:i:s', time());
>
> Date should start from tomorrow:
> $date = date('Y-m-d 00:00:00', strtotime('+1 day'));
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/capacity4dev/capacity4more/pull/463/files#r24338007>.
>


-- 
Sent from My iPhone